src/coreclr/nativeaot/Runtime/MathHelpers.cpp

```diff
@@ -9,27 +9,18 @@
 // Floating point and 64-bit integer math helpers.
 //
 
-FORCEINLINE int64_t FastDbl2Lng(double val)
-{
-#ifdef TARGET_X86
-    return HCCALL1_V(JIT_Dbl2Lng, val);
-#else
-    return((__int64) val);
-#endif
-}
-
 EXTERN_C NATIVEAOT_API uint64_t REDHAWK_CALLCONV RhpDbl2ULng(double val)
 {
     const double two63  = 2147483648.0 * 4294967296.0;
     uint64_t ret;
     if (val < two63)
     {
-        ret = FastDbl2Lng(val);
+        ret = (__int64)(val);
```

Comment:
```suggestion
        ret = (int64_t)(val);
```
Prefer standard C/C++ type names